### PR TITLE
MET-4009 Updated CORS mapping configuration

### PR DESCRIPTION
**Pull request by JoanaCMS**

### CODE REVIEW

File: src/main/java/eu/europeana/metis/sandbox/config/web/MvcConfig.java [hunk 1]
```diff
@@ -31,7 +30,7 @@ public void addFormatters(FormatterRegistry registry) {
 
   @Override
   public void addCorsMappings(CorsRegistry registry) {
-    if (StringUtils.isNotBlank(corsMapping)) {
+    if (corsMapping.length != 0) {
```
Comment (stzanakis) [inline src/main/java/eu/europeana/metis/sandbox/config/web/MvcConfig.java:33]: An empty array probably is allowed in `allowedOrigins` but a null array is not and we are not checking that here.
In fact `corsMapping.length` will fail if the field is `null` before the below `allowedOrigins` execution.
Better use `ArrayUtils.isNotEmpty(array)`

File: src/main/java/eu/europeana/metis/sandbox/config/web/MvcConfig.java [hunk 2]
```diff
@@ -31,7 +30,7 @@ public void addFormatters(FormatterRegistry registry) {
...
       registry.addMapping("/**").allowedOrigins(corsMapping);
```
Comment (stzanakis) [inline src/main/java/eu/europeana/metis/sandbox/config/web/MvcConfig.java:34]: Could you please also add the `.allowedMethods("GET", "HEAD", "POST", "PUT", "DELETE", "OPTIONS")`?
The default are only GET,HEAD,POST.

File: src/main/java/eu/europeana/metis/sandbox/config/web/MvcConfig.java (hunk 1 above)
Comment (JoanaCMS) [reply src/main/java/eu/europeana/metis/sandbox/config/web/MvcConfig.java:33]: Thanks for sharing this, this is definitely a better option, I will add it

File: src/main/java/eu/europeana/metis/sandbox/config/web/MvcConfig.java (hunk 2 above)
Comment (JoanaCMS) [reply src/main/java/eu/europeana/metis/sandbox/config/web/MvcConfig.java:34]: Done

**Pull request merged**